src/modify.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import 'source-map-support/register';
+import { DynamoDBStreamEvent, Context, Callback } from 'aws-lambda';
+import logger from './observability/logger';
+import { eventHandler } from './eventHandler';
+
+const { NODE_ENV, SERVICE, AWS_REGION, AWS_STAGE } = process.env;
+
+logger.debug(
+  `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_STAGE}'\n region: '${AWS_REGION}'\n\n`,
+);
+
+const handler = async (event: DynamoDBStreamEvent, _context: Context, callback: Callback) => {
+  try {
+    logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
+    if (process.env.MODIFY_EVENTS) {
+      await eventHandler(event);
+    } else {
+      logger.debug('Not handling modify events.');
```

Comment:
Wondering if you could remove this if else with something like this and leave the rest.
```
if(!process.env.MODIFY_EVENTS) {
   callback(null,'Not handling modify events')
}
```

Reply:
In fact, would it make sense to extract this check at the eventHandler.ts level to save it being unmarshalled etc

Reply:
This is the handler so the check would happen before the `eventHandler.ts` level, so it wouldn't get unmarshalled.

Reply:
I do like the other suggestion however the tests don't like it for some reason (I think setting environment variables in tests seems a bit touch and go for some reason). Also I think we'll probably just delete this in a future PR to remove the feature toggle. Either that on raise a terraform PR to add the env variable. But I'd rather remove this if so we don't have a random env variable on the lambda forever